src/libs/UnreadIndicatorUpdater/index.js

```diff
@@ -1,27 +1,20 @@
 import _ from 'underscore';
 import Onyx from 'react-native-onyx';
 import ONYXKEYS from '../../ONYXKEYS';
-import CONST from '../../CONST';
 import updateUnread from './updateUnread';
+import * as ReportUtils from '../ReportUtils';
 
-// Stash the unread action counts for each report
-const unreadActionCounts = {};
+// Stash the reports
+const reports = {};
 
 /**
  * Updates the title and favicon of the current browser tab
  * and Mac OS or iOS dock icon with an unread indicator.
  */
 const throttledUpdatePageTitleAndUnreadCount = _.throttle(() => {
-    const totalCount = _.reduce(unreadActionCounts, (total, reportCount) => total + Math.max(reportCount, 0), 0);
-
-    // When we don't have an exact count we just let the user know there's something new
-    if (totalCount === 0 && _.some(unreadActionCounts, count => count === -1)) {
-        updateUnread(1);
-        return;
-    }
-
+    const totalCount = _.reduce(reports, (total, report) => total + (ReportUtils.isUnread(report) ? 1 : 0), 0);
```

Comment:
```suggestion
    const totalCount = _.filter(reports, ReportUtils.isUnread).length
```
NAB, just a suggestion